packages/css/test/index.js

```diff
@@ -226,3 +226,16 @@ test('handles negative margins from scale', () => {
     marginRight: -32,
   })
 })
+
+test('raw values can be passed as second argument', () => {
+  const result = css({
```

Comment:
this can be confusing because emotion allows to pass multiple style objects too and they are not treated differently

Reply:
Yeah, good point. Just wanted to get a feel for this API – probably not the best approach